internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -137,91 +139,245 @@ func TestOLMBundle(t *testing.T) {
 	}
 }
 
-func TestScorecardFuncs(t *testing.T) {
+func TestDescriptors(t *testing.T) {
+	crWithDescriptor := unstructured.Unstructured{
```

Comment:
The way these are set up is fine for now, but these should be refactored in BDD-style tests with ginkgo/gomega soon.

Reply:
In a follow-up